releasenotes/notes/0.25/has-pygments-tester-3fb9f9c34907d45d.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-features:
+features_misc:
```

Comment:
Did we do a `misc` section before?

Reply:
No we didn't. But it seems odd to me to have unclassified contents at the very top of the changelog beneath a heading level 1, rather than level 2 like the rest.